tests/create_private_all/main.tf

```
@@ -0,0 +1,39 @@
+provider "aws" {
+  alias   = "vpc_owner"
+  profile = "awsalternate"
+}
```

Since this is a multi-provider/account test, it must also explicitly define the profile to use for the default aws provider:

```suggestion
provider "aws" {
  profile = "aws"
}

provider "aws" {
  alias   = "vpc_owner"
  profile = "awsalternate"
}
```